app/AndroidManifest.xml

```diff
@@ -3,7 +3,7 @@
           xmlns:tools="http://schemas.android.com/tools"
           package="org.commcare.dalvik"
           android:versionCode="106"
-          android:versionName="2.37.1">
+          android:versionName="2.37.2">
```

Comment:
This version is (I believe) just a string with no semantic value, right? I think we should make it say "Custom Migration Fix 2.37.2"